src/routing_table/mod.rs

```diff
@@ -989,7 +989,7 @@ impl<T: Binary + Clone + Copy + Debug + Default + Hash + Xorable> RoutingTable<T
 
     /// Returns the prefix of the closest non-empty section to `name`, regardless of whether `name`
     /// belongs in that section or not, and the section itself.
-    fn closest_section(&self, name: &T) -> (&Prefix<T>, &BTreeSet<T>) {
+    pub fn closest_section(&self, name: &T) -> (&Prefix<T>, &BTreeSet<T>) {
```

Comment:
as the comment above, get_section may return None, I think this expose is required.